package.json

```diff
@@ -75,6 +75,7 @@
   },
   "dependencies": {
     "attr-accept": "^1.0.3",
+    "html5-file-selector": "^1.0.1",
```

Comment:
please update to "^2.0.1" to reduce the bundle size impact
https://bundlephobia.com/result?p=html5-file-selector@2.0.1